prod/monitoring/installation/helmrelease.yaml

```diff
@@ -0,0 +1,24 @@
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: dnation-kubernetes-monitoring-stack
+  namespace: flux-system
+spec:
+  releaseName: dnation-kubernetes-monitoring-stack
+  chart:
+    spec:
+      chart: dnation-kubernetes-monitoring-stack
```

Comment:
I think we should pin the version here.

Reply:
Yes, we can do that. I would propose to pin the latest version which is `3.6.1`.